dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -254,6 +256,16 @@ interface Props {
 	format: ArticleFormat;
 }
 
+const paddingBody: SerializedStyles = css`
+	padding: ${remSpace[3]};
```

Comment:
I've checked in with the dotcom team and on DCR, px is preferred over rem. As such, we should replace these remSpace values for space and adopt px. This would mean this line would change to 	`padding: ${space[3]}px;` for example.

Reply:
Thanks so much Anna, this has been fixed!